### PR TITLE
Drop support for Symfony <3

### PR DESCRIPTION
**Pull request by lyrixx**

### CODE REVIEW

**Comment by stof** [comment]
should we make a release with all the existing changes before that ?

File: composer.json [hunk 1]
```diff
@@ -17,15 +17,15 @@
     ],
     "require": {
         "php": ">=5.6",
-        "symfony/monolog-bridge": "~2.7|~3.3|~4.0",
-        "symfony/dependency-injection": "~2.7|~3.4.10|^4.0.10",
-        "symfony/config": "~2.7|~3.3|~4.0",
-        "symfony/http-kernel": "~2.7|~3.3|~4.0",
+        "symfony/monolog-bridge": "~3.3|~4.0",
```
Comment (stof) [inline composer.json]: should we change it to `~3.4` too as 3.3 is EOLed (2.8 still has security support for a few months, but 3.3 does not)

Comment (lyrixx) [reply composer.json]: I thought the same thing! But I did not want to bump for the sake of bumping.
But I would go for too

**Comment by lyrixx** [comment]
I don't know. I initially wanted to merge / close as many issue as possible and then make a release.
Before I cleanup this project, the changelog was broken. So making a release without burden could be possible on the very last commit of master f51d377648794e8a8a54f889f17a621911295ce9

But IMHO, it's a bit stupid to do a release, and then to continue to merge new things...

I would go for simplicity: do a release after merge of pending PRs (this one, the one about mailer, and a another couples)

**Comment by lyrixx** [comment]
ping @symfony/deciders

File: composer.json (hunk 1 above)
Comment (fabpot) [reply composer.json]: Let's bump!

**Comment by fabpot** [comment]
I agree, let's do one release with everything, including the bumps for the deps.

**Pull request merged**